content/conditions/diarrhoea/local-header.md

```diff
@@ -0,0 +1,8 @@
+Diarrhoea usually goes away on it’s own within a few days. Make sure you or your child drink plenty of fluids.
+
+- [Adults with diarrhoea](#how-you-can-treat-diarrhoea-yourself)
```

Comment:
Might be worth writing an issue for reviewing the anchor list pattern. Both for here and content store.